test/visual/src/review.js

```diff
@@ -174,10 +174,10 @@ async function run() {
     const response = await fetch('./data.json');
     const data = await response.json();
     const decorator = document.querySelector('sp-story-decorator');
-    const theme = data.meta.theme.split(' ');
-    decorator.theme = theme[0];
-    decorator.color = theme[1];
-    decorator.scale = theme[2];
+    const system = data.meta.system.split(' ');
```

Comment:
This change needs to be mirrored in https://github.com/adobe/spectrum-web-components/blob/main/test/visual/review.js#L181

Reply:
Yes!!

Reply:
@Westbrook Even after updating this I am still seeing the cached `review.js` in the new VRT Urls.
![Screenshot 2024-04-04 at 7 56 25 PM](https://github.com/adobe/spectrum-web-components/assets/8790510/43749f39-e0bb-4385-80c9-e6e35518e4e6)